sig-apps/README.md

```diff
@@ -29,6 +25,56 @@ Covers deploying and operating applications in Kubernetes. We focus on the devel
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-apps)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fapps)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **chart-testing**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-helm/chart-testing/master/OWNERS
+- **chartmuseum**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-helm/chartmuseum/master/OWNERS
+- **charts**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/charts/master/OWNERS
+  - Meetings:
+    - Charts Chat: [Tuesdays at 9:00 PT (Pacific Time)](https://zoom.us/j/166909412) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=9:00&tz=PT%20%28Pacific%20Time%29).
+      - [Meeting notes and Agenda](https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3).
+- **charts-tooling**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-helm/charts-tooling/master/OWNERS
+- **community**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-helm/community/master/OWNERS
+- **examples**
```

Comment:
We should ask the Helm community whether 2 subprojects are enough, or whether approx. 1 subproject per repo is needed, but could we at least group the chart-related and helm-related repos together in this list?

Reply:
To address Brians feedback here's my 2 cents as the groupings can be quite subjective.

* _chart-tooling_ can be grouped with _charts_. That is supporting one off tooling experiments.
* _community_ can be grouped under _helm_ as that is supporting docs.

After that things get a little less clear:

* _monocular_ is worked on by _helm_ and _charts_ folks and leverages code and work from both of those. But, it is it's own application. The same can be said of _chartmuseum_. They could be on their own or they could be lumped under _helm_.
* Depending on how you answer the previous one, _rudder-federation_ and _chart-testing_ could be grouped as individual efforts or separate sub-projects.

A lot of this comes down to what is supporting what and where to break things apart. Below is a tree of how, I think, things relate to each other in the Helm space.

```
                                  +---------+
                                  |  Helm   |
     +----------------------+-----+--------++-------------------+--------------------------+---------------------+
     |                      |              |                    |                          |                     |
     |                      |              |                    |                          |                     |
+----v------+           +---v----+---------+          +---------v----------+        +------v-------+     +-------v-------+
| Community |           | Charts |         |          |  Rudder Federation |        | Chart Museum |     | Chart Testing |
+-----------+    +------+--------+         |          +--------------------+        +--------------+     +---------------+
                 |                   +-----v-----+
                 |                   | Monocular |
                 |                   +-----------+
        +--------v-------+
        | Charts Tooling |
        +----------------+
```

At what level are sub-projects broken apart into smaller bits?

Reply:
I've done the first two groupings.  We can iterate on the rest as needed.  My guess was going to be "everything in kuberentes-helm is part of helm."  Personally I don't see a ton of value in defining turtles all the way down unless there are disjoint sets of maintainers and consumers.

Reply:
I am not opposed to grouping Helm into one project (all the things!) or two projects (Helm code, charts).

In the past, we've tried to make a governance distinction between charts and Helm in order to prevent developers from making ops decisions. So we've talked about them as truly distinct projects. But org-wise, maybe it would actually make sense to group them.

Reply:
@spiffxp If we group things by maintainers you have:

* Helm
* Charts
* Chart Museum

While there are some overlaps those are the most distinct groups of maintainers.

The reason I described turtles all the way down was for clarity. We don't really need to define that for this doc.